src/journal/Journaler.cc

```diff
@@ -47,7 +47,9 @@ Journaler::Threads::Threads(CephContext *cct) {
   thread_pool = new ThreadPool(cct, "Journaler::thread_pool", "tp_journal", 1);
   thread_pool->start();
 
-  work_queue = new ContextWQ("Journaler::work_queue", 60, thread_pool);
+  work_queue = new ContextWQ("Journaler::work_queue",
+			     ceph::make_timespan(60),
```

Comment:
Nit: tabs vs spaces indentation alignment (throughout)

Reply:
@dillaman i am not sure i follow you. i think i am following the coding stlye at 
https://github.com/ceph/ceph/blob/b77da6cb3b4af2b52ca085c12f3c973b243f9619/CodingStyle#L76-L82

and my editor understands and uses

```
-*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

if you could be more specific on what's wrong with the indentation or alignment in my change. i'd be happy to update per your request.

Reply:
You can see the bad indent in your change right above -- I hate mixing tabs and spaces.

Reply:
@dillaman i don't think GitHub reads the editor variables defined in the header. i always try to follow our CodingStyle and also try to avoid fighting with the settings. in my editor, they are aligned vertically.

but i understand your concerns. just replaced the newly introduced tabs with spaces `journal/Journaler.cc` and `librbd/ImageState.cc` in my latest revision. could you take another look?

Reply:
Thanks -- "git" CLI also won't convert like vi/emacs. Can you also get `src/test/journal/RadosTestFixture.cc`?

Reply:
sure. updated and repushed.